src/java/org/apache/cassandra/tools/nodetool/stats/CompactionHistoryHolder.java

```diff
@@ -64,6 +65,7 @@ private static class CompactionHistoryRow implements Comparable<CompactionHistor
             this.bytesIn = bytesIn;
             this.bytesOut = bytesOut;
             this.rowMerged = rowMerged;
+            this.compactionType = compactionType;
```

Comment:
@Maxwell-Guo you have not changed name of this property to `compactionProperties`.

Reply:
thk